controllers/api/api_trusted_controller.py

```diff
@@ -218,3 +218,5 @@ def _process_request(self, request, event_key):
                 match.dirty = True  # This is hacky -fangeugene 2014-10-26
                 matches_to_put.append(match)
         MatchManipulator.createOrUpdate(matches_to_put)
+
+        self.response.out.write(json.dumps({'Success': "Match videos successfully updated"}))
```

Comment:
This is probably what broke the tests.

Added because without some return value, the ajax callback was reporting successes as errors. I can fix the tests to account for that